model/client4.go

```diff
@@ -3939,7 +3941,7 @@ func (c *Client4) GetTermsOfService(etag string) (*TermsOfService, *Response) {
 	defer closeBody(r)
 	return TermsOfServiceFromJson(r.Body), BuildResponse(r)
 }
-
```

Comment:
The build failed because a linebreak is missing here. Run `make gofmt` to check if you fixed it.

Reply:
Done.